### PR TITLE
[cleanups] some cleanups on related subtitles code

### PR DESCRIPTION
**Pull request by CastagnaIT**
## Description

Just some cleanups

## Motivation and context

See jenkins analisys on clang-tidy and cpp warnings

## How has this been tested?

compiled and run a video

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by enen92 — APPROVED** [review]
Thanks much!

**Comment by enen92** [comment]
jenkins build this please

File: xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitleParserSubrip.cpp [hunk 1]
```diff
@@ -65,7 +65,7 @@ bool CDVDSubtitleParserSubrip::Open(CDVDStreamInfo& hints)
           StringUtils::Trim(line);
 
           // empty line, next subtitle is about to start
-          if (line.length() <= 0)
+          if (line.length() == 0)
```
Comment (lrusak) [inline xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitleParserSubrip.cpp:68]:
```suggestion
          if (line.empty())
```

File: xbmc/settings/windows/GUIWindowSettingsScreenCalibration.cpp [hunk 2]
```diff
@@ -309,36 +309,29 @@ void CGUIWindowSettingsScreenCalibration::ResetControls()
...
-                         info.iHeight / 4);
+    pControl->SetLimits(-info.iWidth / 4, -info.iHeight / 4, info.iWidth / 4, info.iHeight / 4);
     auto& size = m_controlsSize[CONTROL_TOP_LEFT];
     pControl->SetHeight(size.first / DEFAULT_GUI_HEIGHT * info.iHeight);
     pControl->SetWidth(size.second / DEFAULT_GUI_WIDTH * info.iWidth);
-    pControl->SetPosition((float)info.Overscan.left,
-                          (float)info.Overscan.top);
-    pControl->SetLocation(info.Overscan.left,
-                          info.Overscan.top, false);
+    pControl->SetPosition((float)info.Overscan.left, (float)info.Overscan.top);
```
Comment (lrusak) [inline xbmc/settings/windows/GUIWindowSettingsScreenCalibration.cpp:321]: c++ style cast

**Pull request merged**